packages/grid/_modules_/grid/components/toolbar/GridFilterToolbarButton.tsx

```diff
@@ -61,7 +77,7 @@ export const GridFilterToolbarButton: React.FC<{}> = () => {
 
   const OpenFilterButtonIcon = apiRef!.current.components!.OpenFilterButtonIcon!;
   return (
-    <Tooltip title={tooltipContentNode} enterDelay={1000}>
+    <Tooltip open title={tooltipContentNode} enterDelay={1000}>
```

Comment:
⚠️

Reply:
The good thing is that the visual tests caught it.